DQMServices/Components/plugins/MEtoEDMConverter.cc

```diff
@@ -92,274 +91,42 @@ void
 MEtoEDMConverter::beginJob()
 {
   // Determine if we are running multithreading asking to the DQMStore. Not to be moved in the ctor
+  DQMStore *dbe = edm::Service<DQMStore>().operator->();
   enableMultiThread_ = dbe->enableMultiThread_;
```

Comment:
isn't `enableMultiThread_` something you wanted to get rid of ?

Reply:
It is. And we are getting rid of it. This PR is a step towards that - or rather to fix the https://github.com/cms-sw/cmssw/issues/22281 with least amount of changes.